LanguageFeatures/Augmentation-libraries/scoping_A01_t07.dart

```diff
@@ -2,18 +2,17 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion If a member declaration needs to reference a static or instance
-/// member declared in another introductory or augmenting declaration of the
-/// same type, it can use `this.name` for instance members an `TypeName.name`
-/// for static members to be absolutely sure. Or it can rely on the default if
-/// `name` is not in the lexical scope at all, in which case it’s interpreted as
-/// `this.name` if it occurs inside a scope where a `this` is available.
+/// @assertion The static and instance member namespaces for an augmented type
+/// or extension declaration include the declarations of all members in the
+/// introductory and augmenting declarations. Identifiers in the bodies of
+/// members are resolved against that complete merged namespace. In other words,
+/// augmentations are applied before identifiers inside members are resolved.
 ///
 /// @description Checks that `name` is interpreted as `this.name` if it occurs
 /// inside a scope where a `this` is available. Test a setter.
```

Comment:
This looks similar to the others, so it should presumably never invoke any of the global declarations.

Reply:
Fixed.